### PR TITLE
SpaceDirectory

### PR DESCRIPTION
**Pull request by dns2utf8**
Migrate to SpaceDirectory and preparations for the next release.

### CODE REVIEW

File: app/src/main/java/ch/fixme/status/Main.java [hunk 1]
```diff
@@ -63,7 +63,7 @@
 public class Main extends Activity {
 
     // API: http://hackerspaces.nl/spaceapi/
-    // http://spaceapi.net
+    // https://spacedirectory.org/
```
Comment (dbrgn) [inline app/src/main/java/ch/fixme/status/Main.java]: I think the first link (at hackerspaces.nl) could be removed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 RELEASE NOTES
 =============
 
+2017-01-XX / 1.8.3
+---
+
+    - Migrate to SpaceDirectory
```
Comment (dbrgn) [inline CHANGELOG.md]: I'd reword this to "Change links from spaceapi.net to spacedirectory.org".

File: app/src/main/java/ch/fixme/status/Main.java (hunk 1 above)
Comment (dns2utf8) [inline app/src/main/java/ch/fixme/status/Main.java]: Done, although the had a cool demo app.

**Comment by rorist** [comment]
Hi, 

I'm not sure I'll be able to release this month, I want to migrate to use Fragments before that and have a better list for hackerspaces with search.

Thanks for the modifications ! \o/

**Comment by dns2utf8** [comment]
Works for me 😄
